README.md

```diff
@@ -0,0 +1,178 @@
+        
+**For all code, bugs, and feature requests related to Data.gov, see the project wide Data.gov [issue tracker](https://github.com/GSA/data.gov/issues).** 
+
+Currently this repository is only used for source version control on the code for the CKAN extension for datagovcatalog features, but you can see all of the Data.gov relevant repos listed in the [GSA Data.gov README file](https://github.com/GSA/data.gov/blob/master/README.md). 
+
+# Requirements
+
+For example, you might want to mention here which versions of CKAN this
+extension works with.
+
+
+Package                                                                | Notes
+---------------------------------------------------------------------- | -------------
+[ckanext-harvest](https://github.com/ckan/ckanext-harvest/)            | This currently has two different version for py2 and py3 until [this PR](https://github.com/ckan/ckanext-harvest/pull/450) is merged upstream
+[ckanext-envvars](https://github.com/okfn/ckanext-envvars)             | This is necessary based on how the testing environment is set up.
+
+
+# Installation
+
+[comment]: <> (Add any additional install steps to the list below.
+   For example installing any non-Python dependencies or adding any required
+   config settings.)
+
+CKAN version | Compatibility
+------------ | -------------
+<=2.7        | no
+2.8          | yes
+2.9          | [complete](https://github.com/GSA/datagov-ckan-multi/issues/570)
+
+To install ckanext-datagovcatalog:
+
+1. Activate your CKAN virtual environment, for example::
+
+     `. /usr/lib/ckan/default/bin/activate`
+
+2. Install the ckanext-datagovcatalog Python package into your virtual environment::
+
+     `pip install ckanext-datagovcatalog`
+
+3. Add ``datagovcatalog`` to the ``ckan.plugins`` setting in your CKAN
+   config file (by default the config file is located at
+   ``/etc/ckan/default/production.ini``).
+
+4. Restart CKAN. For example if you've deployed CKAN with Apache on Ubuntu::
+
+     `sudo service apache2 reload`
+
+
+# Config Settings
+
+[comment]: <> (Document any optional config settings here. For example::
+
+    # Add tracking info on each package for the dataset lists
+    # (optional, default: true).
+    ckanext.datagovcatalog.add_packages_tracking_info = true)
+
+
+# Development Installation
+
+To install ckanext-datagovcatalog for development, activate your CKAN virtualenv and
+do::
+
+    git clone https://github.com/GSA/ckanext-datagovcatalog.git
+    cd ckanext-datagovcatalog
+    python setup.py develop
+    pip install -r requirements.txt
+    pip install -r dev-requirements.txt
+
+**Note: use the py2-equivalents of the requirement files if running on python 2.
+
+# Running the Tests
+
+## Tests
```

Comment:
Remove as unnecessary

Reply:
I don't think there's anything to remove.